pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -298,6 +299,40 @@ def test_iloc_setitem_bool_indexer(self, klass):
         expected = DataFrame({"flag": ["x", "y", "z"], "value": [2, 3, 4]})
         tm.assert_frame_equal(df, expected)
 
+    @pytest.mark.parametrize("dtype", ["int64", "Int64"])
+    def test_setitem_complete_columns_different_dtypes(self, dtype):
```

Comment:
can you make these `test_setitem_null_slice_foo_bar` (or possibly `test_loc_setitem_null_slice_foo_bar` per comment below)

Reply:
Sorry, I don't understand it. I should rename the tests to ``test_loc_setitem_null_slice_foo_bar``?

Reply:
This one would be `test_setitem_null_slice_different_dtypes`, the next one would be `test_setitem_null_slice_single_column_series_value_different_dtype`

basically the goal is to establish a pattern in test naming for indexing tests with test_{method_name}_{key_description}_...

Reply:
Done

Reply:
Thx for the explanation. Could we maybe document this somewhere a bit more extensively?

Reply:
>  we maybe document this somewhere a bit more extensively?

at some point i hope.  the pattern isnt exactly in place yet